### ISSUE TITLE
Should pkg-config be used to find libotr?

### ISSUE THREAD

**Issue by quite** [boothj5/profanity#518]
As per mail exchange with author, the line below is needed if libotr is installed in $HOME. From what I can see, pkg-config is not used for libotr (but indeed for libstrophe) -- so would using it solve this?

```
  ./configure CPPFLAGS=-I$HOME/include LDFLAGS=-L$HOME/lib 
```

**Comment by jubalh** [comment]
@pasis maybe you know more about this? As you are the autoconf master :)

**Comment by pasis** [comment]
Implemented in 671849c71152e7cc9272480ecab6d35f92e199ed. The commit may break builds on systems with incomplete libotr installation. In the case, I'll add a hack to bypass it.
